examples_tests/31.SkinningDataBenchmark/main.cpp

```diff
@@ -109,7 +109,7 @@ int main()
         meshBuffer->setPipeline(std::move(pipeline));
```

Comment:
explain what's going on with `newInputParams`

Reply:
TODO tweak all `newInputParams, blendParams, geometryObject.assemblyParams, rasterParams` to optimize for vertex only workloads

Reply:
Also no fragment shader to be used in benchmark mode.

Reply:
> explain what's going on with `newInputParams`

I just extend vertices by by bone ID attribute here, and it requires updated `SVertexInputParams`

Reply:
ok, but the tweaks for benchmark stuff remain in place.